src/__tests__/develop.spec.js

```diff
@@ -0,0 +1,21 @@
+import GetNumberSum from '../develop.js'
+
+function test() {
+    const testCases = [
+      { input: { a: 1, b: 2 }, expected: 3 },
+      { input: { a: -2, b: 1 }, expected: 0 },
+      { input: { a: 'not a number', b: 'number' }, expected: "Error: Cannot plus non number params" } 
+    ];
+  
+    const testResults = [];
+  
+    testCases.map((testCase) => {
+      const result = GetNumberSum(testCase.input.a, testCase.input.b); 
+      const isPassed = result === testCase.expected; 
```

Comment:
Why we setting isPassed but never use it?
On line 16 we passing same as we already have.